index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello Microverse act </title>
+  <link rel="stylesheet" href="./css/styles.css" />
+</head>
+
+<body>
+<!-- Header Section -->
+<header class="header">
+  <div class="header-container">
+    <a href="#" class="logo animation-rotate">Geegee</a>
+    <nav class="nav-bar">
+      <ul class="navigation-bar nav-menu">
+        <li><a href="#portfolio" class="list-item">Portfolio</a></li>
+        <li><a href="#about" class="list-item">About</a></li>
+        <li><a href="#contact" class="list-item">Contact</a></li>
+      </ul>
+    </nav>
+    <div class="hamburger">
+      <i class="menu-icon fa-solid fa-bars"></i>
+      <i class="cross-icon fa-solid fa-times"></i>
+    </div>
```

Comment:
As the previous reviewer mentioned, please ensure that your mobile menu is visible on the right side of your navbar as per the figma design🙏

Your design:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/37116322/226747882-95953223-97d3-4a08-9038-574f564c893f.png">

Figma design:
<img width="195" alt="image" src="https://user-images.githubusercontent.com/37116322/226747955-30412390-0a19-4613-92e8-5d6001c58842.png">

Reply:
thank you , I fixed it now

Reply:
@Shisui6 Please take a look at it now